app/modules/enduser_setup.c

```diff
@@ -1432,15 +1588,10 @@ static err_t enduser_setup_http_recvcb(void *arg, struct tcp_pcb *http_client, s
           break;
       }
     }
-    else if (strncmp(data + 4, "/generate_204", 13) == 0)
-    {
-      /* Convince Android devices that they have internet access to avoid pesky dialogues. */
-      enduser_setup_http_serve_header(http_client, http_header_204, LITLEN(http_header_204));
-    }
     else
     {
-      ENDUSER_SETUP_DEBUG("serving 404");
-      enduser_setup_http_serve_header(http_client, http_header_404, LITLEN(http_header_404));
+      // All other URLs redirect to / -- this triggers captive portal.
```

Comment:
> redirect to /

Not anymore 😄 You changed this to `http://nodemcu.portal/`.

Reply:
Yes -- I can fix that. It turned out that redirecting to / did not trigger captive portal on MacOs.